core/src/main/java/com/testaem/aem/core/models/impl/MultipleNewsImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.testaem.aem.core.models.impl;
+
+import com.testaem.aem.core.models.MultipleNews;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.Self;
+import org.apache.sling.models.annotations.injectorspecific.SlingObject;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+@Model(
+        adaptables = SlingHttpServletRequest.class,
+        defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL,
+        adapters = MultipleNews.class
+)
+public class MultipleNewsImpl implements MultipleNews {
+
+    private List<String> resources = new ArrayList<>();
+    @SlingObject
+    private ResourceResolver resourceResolver;
+    @Self
+    private SlingHttpServletRequest request;
+
+    private Integer limit;
+
+    @Override
+    public List<String> getResourcesPaths() {
+        int offset = StringUtils.isNumeric(request.getParameter("offset")) ? Integer.parseInt(request.getParameter("offset")) : 0;
```

Comment:
(cosmetic) Try to split long lines. Because you see how it looks like -- inaccurate)) 
A bit hard to understand what relates to what, where's the end of prev statement, and the start of the next. Especially when it's 2am))

![image](https://user-images.githubusercontent.com/48795460/174264926-2d49c339-37e0-4322-a3c3-8c0de55f822e.png)